### ISSUE TITLE
[RTL] The arrow icon in the Tracking Protection popup menu is reversed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6545]
See screenshot:
https://drive.google.com/file/d/1wIrORyiHibQb90FxLqZyjFXebTvIRCl8/view

The arrow icon should be facing left.

Via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634936

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2400)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 228b77b9a89f133417cb748fcf720fb01e6b6f1a

**Comment by SimonBasca** [comment]
Verified fix on v98.0 (8023).
